molgenis/molgenis-emx2
# feat: #3790 add topic select to contact form

What are the main changes you did:
Closes #3790 
Closes #3791 
Closes #3789 

example: http://localhost:3000/catalogue_demo/ssr-catalogue/testNetwork1/cohorts/testCohort1

how to test:
- setup mail service to given contact ( see docs on email service for catalogue)
- make sure to change/select a cohort contact email to a address that is owned by you ( i.e. do not send 'test' email to 'real' cohorts 
- click the contact button
- fill out the form ( with and without filling in topic field
- check email for message topic should be part of subject and message ( if filled in )

todo:
